src/test/app-admin.yaml

```diff
@@ -73,7 +73,7 @@ spec:
     spec:
       serviceAccountName: httpbin
       containers:
-        - image: docker.io/kong/httpbin:0.2.1
+        - image: docker.io/kong/httpbin:0.2.3
```

Comment:
@noahpb from a quick check upstream it looks like this update switched to a nonroot user and also changed some permissions around how this runs. They switched to the `10001` user and group, but I had issues even after switching the security context to run as that user/group. It seemed like if I didn't drop all capabilities it did work, so I suspect there is something more going on. I'd maybe advocate for rolling back the version update temporarily unless you can find a quick fix here? We had an issue with this image before when they changed permissions and I think we temporarily rolled back and/or maybe tried switching to [go-httpbin](https://hub.docker.com/r/mccutchen/go-httpbin)?